### ISSUE TITLE
Training/test splits of the BUCC dataset

### ISSUE THREAD

**Issue by MGithubGA** [google-research/xtreme#13]
Hi,

Congratulations on your paper!

I want to know the training/test splits of the BUCC dataset.
In the paper, it writes "we evaluate representations on the test sets directly", but the training data are renamed as test set at [here](https://github.com/google-research/xtreme/blob/d25b4618bfe1c2a06ca07d7de18ea548f03d020f/scripts/download_data.sh#L132).

`for f in $base_dir/*training*; do mv $f ${f/training/test}; done`

So which spilt is used as the test set of bucc in xtreme? Training set or test set?

Thanks.

**Comment by sebastianruder** [comment]
Hi! Thanks for the question and the close look at the code. Good catch! :)
We used the training set of the BUCC task for evaluation in XTREME as the BUCC test set is private. We might update the evaluation in the future if we get access to the original test data. This is not a problem in practice as no new parameters are learned for this task.
@JunjieHu, do you have anything to add?

**Comment by JunjieHu** [comment]
Thanks for the question! Indeed as Sebastian said, the original test data is private. So we use the released train set as our evaluation test set, and use the released sample set as our [dev set ](https://github.com/google-research/xtreme/blob/d25b4618bfe1c2a06ca07d7de18ea548f03d020f/scripts/download_data.sh#L133) which can be used by the participants, for example to find a threshold for mining the bitext.

**Comment by MGithubGA** [comment]
Thanks for your quick reply! 👍
